.github/workflows/build_and_deploy.yml

```diff
@@ -1,6 +1,8 @@
 # As much as possible, this file should be kept in sync with
 # https://github.com/napari/napari/blob/main/.github/workflows/build_docs.yml
-name: Build PR Docs
+# and https://github.com/napari/napari/blob/main/.github/workflows/deploy_docs.yml
```

Comment:
```suggestion
# This workflow is triggered by  https://github.com/napari/napari/blob/main/.github/workflows/deploy_docs.yml when a merge to `main` occurs in napari/napari
```

Reply:
Hmm so should it *also* be kept in sync with the napari/napari build and deploy docs workflows?

Reply:
Probably worth making an issue over at napari/napari about this.

Reply:
Fixed, thanks!